samples/XCT.Sample/Pages/Views/TabView/DynamicTabsPage.xaml.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Sample.Pages.Views.TabView
+{
+		public partial class DynamicTabsPage
+	{
+		private int TabCounter { get; set; } = 3;
```

Comment:
Should this private field start with underscore?
```suggestion
		private int _tabCounter { get; set; } = 3;
```